confirm_big_donations.js

```diff
@@ -6,8 +6,13 @@ $(document).ready(function() {
    */
   $('form').data('confirmed', false).submit(function(event) {
     var $splitTotal = $('#total-to-split');
-    var $other = $('#amount_other_field');
-    var donationAmount = parseFloat($other.val());
+    var donationAmount = 0;
+    var amountChecked = $('input:radio[name=amount]:checked').val();
+    if (amountChecked.length > 0) {
```

Comment:
Is this testing the string length of val()?  any reason not to just if (amountChecked) {...}, or is there an edge case (in which case, please add comment).